components/SegmentedControl/index.d.ts

```diff
@@ -1,5 +1,7 @@
 import { FC, MouseEventHandler } from 'react';
 
+export type NonEmptyArray<T> = [T, ...T[]];
```

Comment:
We should move this type to a location where it can be easily imported, to avoid duplication